Update 09.peptide-purification.md (#75)
jessegmeyerlab/proteomics-tutorial: content/09.peptide-purification.md

Binding of peptides to this solid phase is based on the hydrophobicity of peptides, mostly due to their peptide backbone, but also due to the presence of amino acid side chains like leucine and phenylalanine. 
To encourage peptides to ‘like’ the stationary phase more than the liquid phase, the peptides are loaded in aqueous solution. 
This will enable washing of the hydrophilic contaminants like salts, small polar buffer molecules, and polar denaturants like urea. 
After washing the bound peptides, they can be eluted by switching the liquid phase to something hydrophobic, which allows the peptides to partition more into the liquid phase and elute from the solid phase.

> ![SPE_v03](https://github.com/jessegmeyerlab/proteomics-tutorial/assets/85298683/a765ff89-00b0-4c1f-8e64-68ac64a05e09)

Reviewer: ```suggestion
```

![**Solid phase extraction (SPE) steps.**

> SPE is a sample preparation technique that uses a solid adsorbent contained most commonly in a cartridge device to selectively adsorb certain molecules from solution. The first step is the conditioning of the cartridge which involves wetting the adsorbent to solvate its functional groups, remove the air present in the column and fill the void spaces with solvent. All this is necessary to produce a suitable environment for adsorption ans thus ensure reproducible interaction with the analytes. After that, the sample is loaded in the cartridge. This can be performed with the aid of positive or negative pressure to ensure a constant flow rate. In this step molecules bind the adsorbent. Next, the column is washed with the mobile phase to eliminate the contaminants. Finally, peptides are eluted in an appropriate buffer solution. 

Reviewer: ```suggestion
SPE is a sample preparation technique that uses a solid adsorbent contained most commonly in a cartridge device to selectively adsorb certain molecules from solution. 
The first step is the conditioning of the cartridge which involves wetting the adsorbent to solvate its functional groups and filling the void spaces with solvent thereby removing any air in the column. 
This is necessary to produce a suitable environment for adsorption and thus ensure reproducible interaction with the analytes. 
After conditioning, the sample is loaded in the cartridge. 
This can be performed with the aid of positive or negative pressure to ensure a constant flow rate. 
In this step molecules bind the adsorbent and interferences pass through. 
Next, the column is washed with the mobile phase to eliminate the contaminants while ensuring the analyte remains bound. 
Finally, peptides are eluted in an appropriate buffer solution with polarity or charge that competes with interaction with the solid phase. 
```

> ](images/SPE_v03.png){#fig:SPE tag="4" width="100%"}

Reviewer: ```suggestion
](images/SPE_v03.png){#fig:SPE tag="3" width="100%"}
```
